inject.js

```diff
@@ -268,9 +268,9 @@ chrome.extension.sendMessage({}, function(response) {
               let ctrl = document.querySelector(`div[data-vscid="${id}"]`)
               if (ctrl) {
                 node.classList.remove('vsc-initialized');
```

Comment:
It's odd that we'd only remove data attr outside of ctrl check. Should the classList remove be outside of the ctrl check as well? Would that break anything?

Reply:
It wouldn't. The class is added at the spot that the data attr is. However, only the attr is checked to see if a video has already been initialized.

Not sure what the class is used for. Unless it's used to style the video in some way, but I can't find the css for that either.

Reply:
For consistency, let's move both to same place -- otherwise, lgtm.